app/components/typography/DescriptionWithChildren.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface DescriptionWithChildrenProps {
+  text: string;
+  lineHeight: string;
+  letterSpacing?: string;
+  children?: React.ReactNode;
+}
+
+const DescriptionWithChildren: React.FC<DescriptionWithChildrenProps> = ({
+  text,
+  children,
+  lineHeight,
+  letterSpacing,
+}) => {
+  return (
+    <h1
+      className={`font-normal text-sm ${lineHeight} ${letterSpacing} text-secondaryColor`}
```

Comment:
String interpolation in class names can result in unintended spacing or issues if one of the values is undefined or empty. Ensure values are safely handled:
```suggestion
      className={`font-normal text-sm ${lineHeight || ""} ${letterSpacing || ""} text-secondaryColor`}
```

Reply:
done